requirements/edx/base.txt

```diff
@@ -524,7 +524,7 @@ edx-search==4.1.3
     #   openedx-forum
 edx-sga==0.25.0
     # via -r requirements/edx/bundled.in
-edx-submissions==3.8.5
+edx-submissions==3.8.6
```

Comment:
Please double check these dependencies

Reply:
For testing purposes, you should be pinning edx-submissions to the branch for the other PRs. Otherwise you're not testing the new changes.